code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -15,94 +15,51 @@ Bonus
 
 //////////////////////////////////////
 */
-/datum/symptom/sensory_restoration
-	name = "Sensory Restoration"
+/datum/symptom/mind_restoration
+	name = "Mind Restoration"
 	stealth = -1
 	resistance = -4
 	stage_speed = -4
 	transmittable = -3
 	level = 5
 	severity = 0
 
-/datum/symptom/sensory_restoration/Activate(var/datum/disease/advance/A)
+/datum/symptom/mind_restoration/Activate(datum/disease/advance/A)
 	..()
 	if(prob(SYMPTOM_ACTIVATION_PROB * 3))
 		var/mob/living/M = A.affected_mob
 		var/datum/reagents/RD = M.reagents
 
-		switch(A.stage)
-			if(2)
-				if(RD.check_and_add("oculine", 10, 10))
-					to_chat(M, "<span class='notice'>Your hearing feels clearer and crisp.</span>")
-			if(3)
-				if(RD.check_and_add("antihol", 10, 10))
-					to_chat(M, "<span class='notice'>You feel sober.</span>")
-				RD.check_and_add("oculine", 10, 10)
-			if(4)
-				if(RD.check_and_add("synaphydramine", 10, 5))
-					to_chat(M, "<span class='notice'>You feel focused.</span>")
-				RD.check_and_add("antihol", 10, 10)
-				RD.check_and_add("oculine", 10, 10)
-			if(5)
-				if(RD.check_and_add("mannitol", 10, 10))
-					to_chat(M, "<span class='notice'>Your mind feels relaxed.</span>")
-				RD.check_and_add("synaphydramine", 10, 5)
-				RD.check_and_add("antihol", 10, 10)
-				RD.check_and_add("oculine", 10, 10)
-
-	return
+		if(A.stage >= 3)
+			M.AdjustSlur(-2)
+			M.AdjustDrunk(-4)
+			M.reagents.remove_all_type(/datum/reagent/consumable/ethanol, 3, 0, 1)
+		if(A.stage >= 4)
+			M.AdjustDrowsy(-2)
+			if(RD.has_reagent("lsd"))
+				RD.remove_reagent("lsd", 5)
+			if(RD.has_reagent("histamine"))
+				RD.remove_reagent("histamine", 5)
+			M.AdjustHallucinate(-10)
+		if(A.stage >= 5)
+			RD.check_and_add("mannitol", 10, 10)
 
-/*
-//////////////////////////////////////
-Sensory-Destruction
-	noticable.
-	Lowers resistance
-	Decreases stage speed tremendously.
-	Decreases transmittablity tremendously.
-	the drugs hit them so hard they have to focus on not dying
-
-Bonus
-	The body generates Sensory destructive chemicals.
-	You cannot taste anything anymore.
-	ethanol for extremely drunk victim
-	lsd to break the mind
-	impedrezene to ruin the brain
-
-//////////////////////////////////////
-*/
-/datum/symptom/sensory_destruction
-	name = "Sensory destruction"
+/datum/symptom/sensory_restoration
+	name = "Sensory Restoration"
 	stealth = -1
-	resistance = -2
-	stage_speed = -3
+	resistance = -3
+	stage_speed = -2
 	transmittable = -4
-	level = 6
-	severity = 5
+	level = 4
 
-/datum/symptom/sensory_destruction/Activate(var/datum/disease/advance/A)
+/datum/symptom/sensory_restoration/Activate(datum/disease/advance/A)
 	..()
-	if(prob(SYMPTOM_ACTIVATION_PROB))
+	if(prob(SYMPTOM_ACTIVATION_PROB * 5))
```

Comment:
Again another gameplay change here, please note in your CL that the symptom effect chance has been increased for new sensory restoration.

Reply:
As noted in the changelog, this is renaming the vision restoration to sensory restoration (as the "vision" one technically also restored ears). 

There is fundamentally no change made other than a name change. I also documented that sensory destruction was removed.

Reply:
Your CL is still one long collection of run-ons that could do for re-write. Only criticism that still stands for me.